cmd/rook/ceph/cleanup.go

```diff
@@ -55,65 +53,18 @@ func startCleanUp(cmd *cobra.Command, args []string) error {
 	logger.Info("starting cluster clean up")
 	// Delete dataDirHostPath
 	if dataDirHostPath != "" {
-		cleanupDirPath := path.Join(dataDirHostPath, namespaceDir)
-		if err := os.RemoveAll(cleanupDirPath); err != nil {
-			logger.Errorf("failed to clean up %q directory. %v", cleanupDirPath, err)
-		} else {
-			logger.Infof("successfully cleaned up %q directory", cleanupDirPath)
-		}
-		// Remove all the mon directories.
-		cleanMonDirs()
-	}
-
-	return nil
-}
-
-func cleanMonDirs() {
-	monDirs, err := filepath.Glob(path.Join(dataDirHostPath, "mon-*"))
-	if err != nil {
-		logger.Errorf("failed to find the mon directories on the dataDirHostPath %q. %v", dataDirHostPath, err)
-		return
+		// Remove both dataDirHostPath and monitor store
+		cleanup.StartMiscCleanup(namespaceDir, dataDirHostPath, monSecret)
```

Comment:
I thought we could rename this Misc function? It's just the host path cleanup, right?